Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -215,6 +215,8 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
     double minScaleFactor() const final;
     double maxScaleFactor() const final;
 
+    bool pluginFitsContent() const final;
```

Comment:
Should we call this `shouldSizeToFitContent`?

Reply:
Sure!

Reply:
Changed in https://github.com/WebKit/WebKit/commit/e6abaffdcea753bb1711d92bc067f42526af4c96